src/decision/dMCDA.jl

```diff
@@ -45,6 +49,53 @@ function mcda_methods()
     ]
 end
 
+"""
+    mcda_method_names()::Vector{String}
+
+List of MCDA method names.
+"""
+function mcda_method_names()::Vector{String}
+    return [
+        "Cocoso",
+        "Mairca",
+        "Moora",
+        "Piv",
+        "Vikor"
+    ]
+end
```

Comment:
I thought this was already implemented elsewhere?

Reply:
https://github.com/open-AIMS/ADRIA.jl/blob/3c2f7cbba94fe821b4dafb24ba41632778f86127/src/decision/dMCDA.jl#L38-L46

Suggest:

```julia
"""
    mcda_method_names()::Vector{String}

List of MCDA method names by order defined in `ADRIA.decision.mcda_methods()`.
"""
function mcda_method_names()::Vector{String}
    return getindex.(split.(string.(ADRIA.decision.mcda_methods()), "."), 2)
end
```

The above returns:

```
5-element Vector{SubString{String}}:
 "COCOSO"
 "MAIRCA"
 "MOORA"
 "PIV"
 "VIKOR"
```